File: python/programmers/brute_force/최소직사각형.py
def solution(sizes):
    answer = 0
    min_big = 0
    max_big = 0
    for s in sizes:
        if s[0] < s[1]:
            if min_big < s[0]:
                min_big = s[0]
            if max_big < s[1]:
                max_big = s[1]
        else:
            if min_big < s[1]:
                min_big = s[1]
            if max_big < s[0]:
                max_big = s[0]
    answer = min_big * max_big

    return answer

File: java/src/main/java/ps_traning/programmers/level_1/최소직사각형.java
package ps_traning.programmers.level_1;

public class 최소직사각형 {
    public int solution(int[][] sizes) {
        int answer = 0;
        int bMax = 0;
        int sMax = 0;
        for (int[] size : sizes) {
            bMax = Math.max(bMax, Math.max(size[0], size[1]));
            sMax = Math.max(sMax, Math.min(size[0], size[1]));
        }
        answer = bMax * sMax;
        return answer;
    }
}
